lib/__test__/rule-list.test.js

```diff
@@ -387,16 +387,16 @@ describe('rules', () => {
     ])
 
     suite('multiplying coefficients', rules.MULTIPLY_COEFFICIENTS, [
-        ['x^2 * y^2', '(1 * 1) (x^2 * y^2)'],
-        ['x^2y^2z^2 * 2x^2', '(1 * 2) (x^2 * y^2 * z^2 * x^2)'],
-        ['3x^2 * x^2', '(3 * 1) (x^2 * x^2)'],
-        ['x^3 * y^2', '(1 * 1) (x^3 * y^2)'],
+        ['x^2 * y^2', 'x^2 * y^2'],
```

Comment:
This might need tweak when it gets moved over to `mathsteps` b/c this should not be a change step b/c nothing changed but right now as long as `rewriteNode` returns a non-null value we assume that a change has occurred.  We can take care of tweaking like that in `mathsteps` though.